containers/javascript-node-lts/.devcontainer/Dockerfile

```diff
@@ -5,33 +5,22 @@
 
 FROM node:lts
 
-# Configure apt
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update \
-    && apt-get -y install --no-install-recommends apt-utils 2>&1
-
-# Verify git and needed tools are installed
-RUN apt-get install -y git procps
+ENV DEBIAN_FRONTEND=dialog
+ENV SHELL /bin/bash
 
-# Remove outdated yarn from /opt and install via package 
-# so it can be easily updated via apt-get upgrade yarn
-RUN rm -rf /opt/yarn-* \
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils 2>&1 \
+    && apt-get install -y git procps \
+    && rm -rf /opt/yarn-* \
```

Comment:
Merging all the `RUN` commands into a single layer will be a good optimization. But, As mentioned earlier we are using a template for Dockerfiles, which means few layers are in common among different dev containers so this means we are actually optimizing the docker build process by dividing them into different `RUN` layers wherever necessary for better caching. So dev containers with similar layers will build faster due to cache but in your case and running all commands in a single intermediate container has least possibility to match other dev containers so making the build process slow.

Reply:
I think as long as we can make it clear with comments and the "&&" being before the line so people can see that it is a continuation, we are probably okay here.  The main concern is getting things so it is "cut-and-pasteable" into another definition. That was my main comment on that point [here](https://github.com/microsoft/vscode-remote-try-cpp/pull/3#issuecomment-497779354).  It makes it clearer that the commands are a continuation of a broader command.